_pages/openid_connect.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: OpenID Connect
+permalink: /openid-connect
```

Comment:
Also, the way [Federalist](https://federalist.18f.gov) works (our homegrown pages server), all internal links need a trailing `/`, so if you _do_ use a permalink, need to add that. Just FYI.